versioned_docs/version-2.7/how-to-guides/new-user-guides/helm-charts-in-rancher/create-apps.md

```diff
@@ -109,20 +109,24 @@ This reference contains variables that you can use in `questions.yml` nested und
 | 	label             | string  | true      |  Define the UI label. |
 | 	description       | string  | false      |  Specify the description of the variable.|
 | 	type              | string  | false      |  Default to `string` if not specified (current supported types are string, multiline, boolean, int, enum, password, storageclass, hostname, pvc, and secret).|
-| 	required          | bool    | false      |  Define if the variable is required or not (true \| false)|
+
```

Comment:
The table broke. I suspect that this blank newline is the culprit.

![image](https://github.com/rancher/rancher-docs/assets/7773090/eb36e54c-814f-4bf3-b720-96aac86aa007)